firmware/Makefile

```diff
@@ -343,9 +344,13 @@ installRfCatDonsDongle: RfCatDons.hex
 
 installRfCatDonsDongleCCBootloader: RfCatDonsCCBootloader.hex
 	if [ ! -c /dev/RFCAT_BL_D ] ; then ../rfcat --bootloader --force && sleep 1 ; fi ;
-	rfcat_bootloader /dev/RFCAT_BL_D erase_all
-	rfcat_bootloader /dev/RFCAT_BL_D download bins/RfCatDonsCCBootloader.hex
-	rfcat_bootloader /dev/RFCAT_BL_D verify bins/RfCatDonsCCBootloader.hex && rfcat_bootloader /dev/RFCAT_BL_D run
+#	rfcat_bootloader /dev/RFCAT_BL_D erase_all
+#	rfcat_bootloader /dev/RFCAT_BL_D download bins/RfCatDonsCCBootloader.hex
+#	rfcat_bootloader /dev/RFCAT_BL_D verify bins/RfCatDonsCCBootloader.hex && rfcat_bootloader /dev/RFCAT_BL_D run
+	./rfcat_bootloader /dev/ttyACM0 erase_all
```

Comment:
please install the udev config file as instructed in README.  this change will is unnecessary

Reply:
Ooops, accidentally included this! Will remove it